400-claims.md

```diff
@@ -8,11 +8,15 @@ weight: 400
 
 This specification describes the CNAB Claims system. This is not part of the [CNAB Core](100-CNAB.md) specification, but is a specification describing how records of CNAB installations may be formatted for storage. Implementations of CNAB Core can meet the CNAB Core specification without implementing this specification. Implementations that support claims MAY state that they comply with CNAB Claims 1.0-WD.
 
-A _claim_ (or _claim receipt_) is a record of a CNAB installation. This specification defines the format of a claim, and describes certain necessary behaviors of a system that implements claims.
+In CNAB, an _installation_ is a particular installed instance of a CNAB bundle. For example, if a CNAB bundled named `myApp` is installed in two places, we say there are _two installations of `myApp`_.
+
+An installation MAY change over time, as a particular bundle is installed, then upgraded. In CNAB, each time a mutating (destructive) operation is performed (Such as `install`, `upgrade`, or custom operations that are not read-only), the claim gets a new _revision_. A revision is a unique identifier that identifies the combination of an _installation_ and a modification of that installation. For example, when a CNAB bundle is installed, the initial installation will have an initial revision ID. When that installation is upgraded, it will have a new revision ID. How revisions are used is outside of the scope of this document, but it is safe to assume that if a revision ID has changed, one or more artifacts owned by the installation has also been changed.
+
+A _claim_ (or _claim receipt_) is a record of a CNAB _installation_. This specification defines the format of a claim, and describes certain necessary behaviors of a system that implements claims.
```

Comment:
I think it might be simpler to only refer to claims as claims rather than introducing _claim receipt_. A receipt carries some additional meaning that may not be applicable to how we treat claims (ie. in real life you get a receipt _after_ completing a transaction, instead of being required to produce one before starting a transaction).

```suggestion
A _claim_ is a record of a CNAB _installation_. This specification defines the format of a claim, and describes certain necessary behaviors of a system that implements claims.
```